synapse/models/geopol.py

```diff
@@ -175,6 +176,8 @@ def getModelDefs(self):
                             'doc': 'The political party of the person who held office during the term.'}),
                     )),
                     ('pol:candidate', {}, (
+                        ('id', ('str', {}), {
```

Comment:
no options on this? seems like lower and onespace would be worthy?

Reply:
nope, the IDs can be arbitrary and should match the provided ID ( the one in the test is for Harris for president )